swap_meet/vendor.py

```diff
@@ -1,2 +1,213 @@
+import uuid
+from swap_meet.item import Item
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        # inventory default []
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+        
+    #help function to check inventory empty
+    def check_inventory_empty(self):
+        if len(self.inventory) == 0:
+            return None
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return None
+    
+    def get_by_id(self, id):
+        for item in self.inventory:
+            if item.id == id:
+                return item
+        return None
+    
+    def swap_items(self, b_vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in b_vendor.inventory :
+            return False
+        else:
+            self.remove(my_item)
```

Comment:
good use of making use of existing vendor functions, rather than re-implementing that logic here again!